templates/blocks/take_action_boxout.twig

```diff
@@ -14,8 +14,8 @@
 				href="{{ boxout.link|default('#') }}" {{ boxout.new_tab and boxout.link ? 'target="_blank"' }}
 			></a>
 			<img src={{ boxout.image }} />
```

Comment:
We should probably add the `alt` attribute here, and provide an `srcset` so we don't have to serve oversized images.

Reply:
Not necessarily here in this PR, but maybe it's nice to have out of the way before doing the AB test.

Reply:
Just added the alt text 🙂